modules/notifications/notifications.php

```diff
@@ -678,35 +680,38 @@ function notification_status_change( $new_status, $old_status, $post ) {
 			$body .= sprintf( __( 'Add editorial comment: %s', 'edit-flow' ), $edit_link . '#editorialcomments/add' ) . "\r\n";
 			$body .= sprintf( __( 'Edit: %s', 'edit-flow' ), $edit_link ) . "\r\n";
 			$body .= sprintf( __( 'View: %s', 'edit-flow' ), $view_link ) . "\r\n";
-				
+
 			$body .= $this->get_notification_footer($post);
-			
+
 			$this->send_email( 'status-change', $post, $subject, $body );
+
+			if ( 'on' === $this->module->options->send_to_slack )
```

Comment:
@hanifn Minor, can you change any new `if`s to use braces? I see the plugin is doing braceless `if`s all over the place but I'd prefer any new code avoid that.

```suggestion
			if ( 'on' === $this->module->options->send_to_slack ) {
			    $this->send_to_slack( $body );
			}
```

Reply:
I actually prefer braces myself as it's clearer to read but I do want to keep the code consistent in style. I'm not sure if changing that is a good idea if we're not planning to change existing code as well. Perhaps it's time to write a style guide for this repo? 🤔

Reply:
I get keeping consistency with a codebase's style, but I think this may be a good time to start changing it.

A style guide would be great, as well as upgrading our PHPCS rules. @ingeniumed and I had a discussion about this [in JS linting rules](https://github.com/Automattic/Edit-Flow/pull/723/files#r1608821536), where we decided we would avoid massive rewrites to meet modern linting rules, but instead incrementally improve code as we encounter it. I think we should do the same here. We can waterfall from the top to the bottom of the list depending on what makes sense:

1. Fix a problem in code for the whole plugin, or if that's out of scope
2. Fix the problem for the whole file, or at least
3. Improve the code for changed functions, or if that's too risky
4. Ensure new code follows modern standards

I think we should encode what is "a problem in code" with PHPCS rules and linting, but for now I think conditionals without braces [are a Bad Practice](https://www.codecentric.de/wissens-hub/blog/curly-braces) enough to ensure we don't write new code this way. Same with `array()` versus `[]` literals, there's no great reason to adopt 2010 PHP coding standards into new features. We may never do a rewrite of some older features to be safe, but we can at least ensure new code is up to date.

Reply:
Agreed on the above. We should approach each PR with the goal of modernizing every file we touch, which would mean:

- Lint the code if it's in PHP
- Modernize the PHP code
- Remove the JQuery pieces

Reply:
Understood 👍 
I also agree with what @alecgeatches said and if we're already planning to overhaul everything that I think this is a fine way to start.